### ISSUE TITLE
Refactor the variables containing the names of the JSON files

### ISSUE THREAD

**Issue by dEhiN** [dEhiN/Demord#36]
Last two comments taken from issue #31 that prompted the creation of this issue:

```
Copy-pasting the original tasks written up in a comment from last week outlining what needs to be done for this issue. There were 3 tasks and the first one has already been completed. The remaining two are:

2. Extrapolate the location of the default startup data JSON file to be stored in only one place - the main comp_start app

3. Change any places where the default startup data JSON file is handled to use the new location for grabbing the actual filename
```

```
While starting to work on the second task, realised that because the branch for this issue is based on the branch features/startup_data_modifier_tool, certain newer code changes made in the branch dEhiN/issue4 will need to be merged in. Specifically, in the other branch, the function is_production was moved from the main comp_start.py script to the module helper.

Extrapolating the location of the default startup data JSON file, or even the in-use startup data JSON file, to be stored in only one place should then be placed in the same module for consistency. However, the branch for this issue doesn't have that code change, so it would be difficult to make the same change.

Will move the 2 tasks left to a new enhancement issue for later and will resolve this issue.
```

This means that both the variable `json_filename` - in the script `comp_start.py` - and the variable `file_name` - in the function `generate_default_startup_data`, in the module `data_generate` - will need to be refactored so they both grab the appropriate values from one location.

The idea will be similar to the current function `is_production`, in the `helper` module, which grabs the value of a global variable meant to specify if the Python tool is being run in production or not. In a similar way, rather than having more than one location to keep track of for any possible future changes to the JSON filenames, it will be better to have one location, such as one or more global variables, and have a function that returns the value of that variable or those variables.

**Comment by dEhiN** [comment]
Created branch dEhiN/issue36 to work on this.

**Comment by dEhiN** [comment]
Through commit e880881 (mentioned above), a helper function was added to the _helper_ module called _get_startup_filename_ which allows for the extrapolation of the location of the default startup data file (and the in-use startup data file). This function isn't in the main module but it still accomplishes the same purpose - to have one location where the file names are stored and can be edited for future use. The only thing left is to update any references in the code to these 2 files to use this function instead.

**Comment by dEhiN** [comment]
Closing issue as both tasks mentioned above are completed and this issue is resolved. The changes have been merged back to the branch for issue #4.
